web/chat/rules.react.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import * as SimpleMarkdown from 'simple-markdown';
+
+export default function rules() {
+  return {
+    // We are using default simple-markdown rules
+    // For more details, look at native/components/rules.react
+    autolink: SimpleMarkdown.defaultRules.autolink,
+    link: SimpleMarkdown.defaultRules.link,
```

Comment:
We should make sure links open in a new tab. I think we'll need to customize the `render` function of `link` to match the old `linkifyComponentDecorator`